packages/katello/katello-repos/katello-repos.spec

```diff
@@ -68,6 +68,9 @@ rm -rf %{buildroot}
 %config %{repo_dir}/*.repo
 
 %changelog
+* Fri Aug 25 2023 David Moore <damoore@redhat.com> - 4.10-0.2.nightly
```

Comment:
Just need to edit the versions in your comments on each of the files (and then I think it can go out of draft!)

Reply:
which versions? -0.3.nightly?

Reply:
you need to match the version you're bumping up to. So it'll be 4.11-0.1.nightly.